pkg/cloud/azure/storageconnection.go

```diff
@@ -45,24 +46,32 @@ func (sc *StorageConnection) getBlobURLTemplate() string {
 	return "https://%s.blob.core.windows.net/%s"
 }
 
-func (sc *StorageConnection) DownloadBlob(blobName string, client *azblob.Client, ctx context.Context) ([]byte, error) {
```

Comment:
@Sean-Holcomb I found no other references to `DownloadBlob()` so I've deleted it. Let me know if we need to bring it back.